network-upgrades/mainnet-upgrades/shanghai.md

```diff
@@ -1,19 +1,24 @@
 ## Shanghai Network Upgrade Specification
 
 ### Included EIPs
-Specifies changes included in the Network Upgrade.
+Changes included in the Network Upgrade.
 
 * [EIP-3651: Warm COINBASE](https://eips.ethereum.org/EIPS/eip-3651)
 * [EIP-3855: PUSH0 instruction](https://eips.ethereum.org/EIPS/eip-3855)
 * [EIP-3860: Limit and meter initcode](https://eips.ethereum.org/EIPS/eip-3860)
 * [EIP-4895: Beacon chain push withdrawals as operations](https://eips.ethereum.org/EIPS/eip-4895)
 
 ### EIPs Considered for Inclusion
-Specifies changes potentially included in the Network Upgrade, pending successful deployment on Client Integration Testnets.
```

Comment:
i think the change in definition warrants a reclassification of those EIPs already in this category, for example i think transient storage was CFI for shanghai meaning that assuming it did not delay shanghai or present issues during testing it would be included, but this change makes that far less clear

Reply:
@moodysalem in the interest of providing an accurate view of where things are at sooner rather than later, I think I'd merge it as is. On the next ACD, I expect we'll have a clear list of what's included in Shanghai and a rough idea of what to do about other EIPs. It makes sense to revamp CFI for the next fork (as we do a bunch of other process changes as well). For 1153 specifically, I don't think it should change what we do: next step is to get it on devnets along with everything else!